rubygem-azure/rubygem-azure.spec

```diff
@@ -0,0 +1,104 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Generated from azure-0.7.5.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name azure
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.7.5
+Release: 1%{?dist}
+Summary: Official ruby client library to consume Microsoft Azure services
```

Comment:
I think you've got summary/description swapped, they should be the other way around.